C/CasADi/build_tarballs.jl

```diff
@@ -0,0 +1,79 @@
+using BinaryBuilder, Pkg
+
+name = "CasADi"
+
+version = v"3.5.5"
+
+sources = [
+    GitSource(
+        "https://github.com/casadi/casadi.git",
+        "fadc86444f3c7ab824dc3f2d91d4c0cfe7f9dad5",
+    ),
+    DirectorySource("./bundled"),
+]
+
+script = raw"""
+cd $WORKSPACE/srcdir/casadi
+install_license LICENSE.txt
+
+mkdir -p build
+cd build
+
+export CXXFLAGS="-fPIC -std=c++11"
+export CFLAGS="${CFLAGS} -fPIC"
+
+cmake -DCMAKE_INSTALL_PREFIX=${prefix} \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DWITH_IPOPT=ON \
+    -DWITH_EXAMPLES=OFF \
+    -DWITH_DEEPBIND=OFF \
+    ..
+
+if [[ "${target}" == *-linux-* ]]; then
+        make -j ${nproc}
+else
+    cmake --build . --config Release --parallel
+fi
```

Comment:
I'm really not convinced there is much difference with
```suggestion
make -j ${nproc}
```